.github/workflows/pytest.yml

```diff
@@ -120,9 +120,7 @@ jobs:
           path: .coverage
 
   ci-integration:
-    runs-on:
-      group: large-runner-group
-      labels: ubuntu-latest-4core
+    runs-on: ubuntu-latest
```

Comment:
See [this pytest action run](https://github.com/catalyst-cooperative/pudl/actions/runs/11204094947) for confirmation that the larger paid runner is no longer necessary.